components/ModEditHealthcareProfessionalSection.vue

```diff
@@ -0,0 +1,282 @@
+            >
+                {{ $t("modHealthcareProfessionalSection.selectLocales") }}
+            </label>
+            <select
+                id="locales"
+                v-model="sectionFields.healthcareProfessionalLocales"
+                data-testid="mod-healthcare-professional-section-spoken-locales"
+                name="Locales"
+                multiple
+                class="mb-5 px-3 py-3.5 w-96 h-32 bg-secondary-bg rounded-lg border border-primary-text-muted
+                            text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
+            >
+                <option
+                    v-for="(locale, index) in localeList"
+                    :key="`${locale}-${index}`"
+                    :value="locale"
+                >
+                    {{ locale }}
+                </option>
+            </select>
+        </div>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import { onMounted } from 'vue'
+import { ModerationScreen, useModerationScreenStore } from '~/stores/moderationScreenStore'
+import { useHealthcareProfessionalsStore } from '~/stores/healthcareProfessionalsStore'
+import { Locale, type LocalizedNameInput, Insurance, Degree, Specialty } from '~/typedefs/gqlTypes'
+import { multiSelectWithoutKeyboard } from '~/utils/multiSelectWithoutKeyboard'
+import SVGTrashCan from '~/assets/icons/trash-can.svg'
+import SVGProfileIcon from '~/assets/icons/profile-icon.svg'
+
+const moderationScreenStore = useModerationScreenStore()
+const healthcareProfessionalsStore = useHealthcareProfessionalsStore()
+const sectionFields = healthcareProfessionalsStore.healthcareProfessionalSectionFields
+const professionalNameArray = sectionFields.healthcareProfessionalNameArray
+const isEditSubmissionScreen = moderationScreenStore.activeScreen === ModerationScreen.EditSubmission
+
+const insuranceList = Object.values(Insurance) as Insurance[]
+const selectedInsurance = (option: HTMLOptionElement): Insurance => option.value as Insurance
+const degreeList = Object.values(Degree) as Degree[]
+const selectedDegree = (option: HTMLOptionElement): Degree => option.value as Degree
+const specialtyList = Object.values(Specialty) as Specialty[]
+const selectedSpecialty = (option: HTMLOptionElement): Specialty => option.value as Specialty
+const localeList = Object.values(Locale) as Locale[]
+const selectedLocale = (option: HTMLOptionElement): Locale => option.value as Locale
+
+const handleLocalizedName = () => {
+    const localizedNameToAdd: LocalizedNameInput = {
+        firstName: sectionFields.localizedFirstName.value,
+        lastName: sectionFields.localizedLastName.value,
+        locale: sectionFields.nameLocale.value || Locale.EnUs,
+        middleName: sectionFields.localizedMiddleName.value
+    }
+
+    if (localizedNameToAdd.firstName
+      && localizedNameToAdd.lastName
+      && localizedNameToAdd.firstName.length > 1
```

Comment:
```suggestion
      && localizedNameToAdd.firstName.length
```
for checking existence, you can simplify and remove the right side